rules/windows/persistence_app_compat_shim.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the installation of custom Application Compatibility Shim databases. This Windows functionality has been
+abused by attackers to stealthily gain persistence and arbitrary code execution in legitimate Windows processes.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Installation of Custom Shim Databases"
+risk_score = 21
+rule_id = "c5ce48a6-7f57-4ee8-9313-3d0024caee10"
+severity = "medium"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+// dependent on wildcard for registry.value
+
+sequence by process.entity_id, process.pid, process.name with maxspan=5m
+  [process where event.type in ("start", "process_started") and
+     not (process.name == "sdbinst.exe" and process.parent.name == "msiexec.exe")]
+  [registry where event.type in ("creation", "change") and
+     registry.key == "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\Custom" and
```

Comment:
is this `key` or `path`?
can you also add `registry.hive`

Reply:
So, originally, the rule looked for

```sql
registry where key_path == "\\REGISTRY\\MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\Custom\\*.sdb"
```

I broke it into key and value, but to avoid a leading wildcard, `registry.path` would actually be better here. I will update